src/svyby.jl

```diff
@@ -37,3 +37,14 @@ function svyby(formula::Symbol, by::Symbol, design::AbstractSurveyDesign, func::
     gdf = groupby(design.data, by)
     return combine(gdf, [formula, :weights] => ((a, b) -> func(a, design, b, params...)) => AsTable)
 end
+
+"""
+    svyby(formula, by, design, function, params)
```

Comment:
The signature needs to be updated as well, `params` doesn't exist anymore.

Reply:
Okay

Reply:
I have edited the docstring in new commit